Resources/Prototypes/Entities/Objects/Devices/Syndicate_Gadgets/boxed_kits.yml

```diff
@@ -0,0 +1,61 @@
+#base syndicate boxes
+- type: entity
+  parent: BoxBase
+  id: SuspiciousBoxBase
+  name: suspicious box
+  description: A box coloured in syndicate red-and-black style.
+  components:
+  - type: Sprite
+    netsync: false
+    sprite: Objects/Storage/boxes.rsi
+    state: box_of_doom
+  - type: Storage
+    capacity: 15
+  - type: Item
+    sprite: Objects/Storage/boxes.rsi
+    state: box_of_doom
+    size: 20
+    HeldPrefix: box
+
+# The box size values may change depending on what is stored in them
+# But if you want to balance a high-risk item by having its box be nigh un-conceallable then you might want to consider a bigger box  
+    
+- type: entity
+  parent: BoxBase
+  id: BigSuspiciousBoxBase
+  name: heavy suspicious box
+  description: A hefty box coloured in syndicate red-and-black style
```

Comment:
As much as I'd prefer it to not be the case we use US spelling here.

Reply:
Abominable. Will be fixed.